Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:lunar as builder
+
+WORKDIR /work
+
+RUN apt update
+RUN apt -y install gcc-12 g++-12 curl make git libjpeg-dev binutils-dev libicu-dev
+
+RUN ln -s /usr/bin/g++-12 /usr/bin/g++
+RUN ln -s /usr/bin/gcc-12 /usr/bin/gcc
+RUN ln -s /usr/bin/gcc-12 /usr/bin/cc
+RUN ln -s /usr/bin/gcc-ar-12 /usr/bin/gcc-ar
+RUN ln -s /usr/bin/gcc-nm-12 /usr/bin/gcc-nm
+
+RUN curl https://sh.rustup.rs | sh -s -- -y
+
+COPY . .
+WORKDIR /work/yanu-cli
```

Comment:
isn't this supposed to be `/work/crates/yanu-cli`? is this alright?

Reply:
yeah, makes sense - will change that soon

Reply:
It's already tweaked by you